### PR TITLE
MetaMetrics: Identify 'nft_autodetection_enabled' &  'opensea_api_enabled'

### PR DESCRIPTION
**Pull request by digiwand**
## Explanation

Add two new user traits to Segment:
- nft_autodetection_enabled - NFT Autodetection Enabled
- opensea_api_enabled - OpenSeas API Enabled

## More information

Progresses https://github.com/MetaMask/metamask-extension/issues/13477

### CODE REVIEW

**Review by segun — DISMISSED** [review]
LGTM

**Review by ryanml — DISMISSED** [review]

File: shared/constants/metametrics.js [hunk 1]
```diff
@@ -160,8 +160,12 @@
  * type is changed we identify the ledger_connection_type trait
  * @property {string} [NETWORKS_ADDED] - when user modifies networks we
  * identify the networks_added trait
+ * @property {string} [NFT_AUTODETECTION_ENABLED] - when Autodetect NFTs
```
Comment (brad-decker) [inline shared/constants/metametrics.js:163]: can you change the string value of the new entries to literals, aka 'nft_autodetection_enabled' which is being done in my new PR as well

**Review by brad-decker — CHANGES_REQUESTED** [review]

File: shared/constants/metametrics.js (hunk 1 above)
Comment (brad-decker) [reply shared/constants/metametrics.js:163]: Only on the new entries, even though thats counter-intuitive, my PR will get the old ones.

Comment (digiwand) [reply shared/constants/metametrics.js]: @brad-decker Good with me. Fixed here: [2588b23](https://github.com/MetaMask/metamask-extension/pull/14367/commits/2588b23a5673ee1000bd8664708c5ef4b508e723)

**Review by brad-decker — DISMISSED** [review]

**Review by segun — DISMISSED** [review]

**Review by jpuri — DISMISSED** [review]

**Review by segun — DISMISSED** [review]

**Review by ryanml — APPROVED** [review]

**Review by jpuri — APPROVED** [review]

**Pull request merged**